apps/wallet-mobile/src/yoroi-wallets/hooks/index.ts

```diff
@@ -1,7 +1,7 @@
 /* eslint-disable @typescript-eslint/no-explicit-any */
 import {walletChecksum} from '@emurgo/cip4-js'
 import {Certificate} from '@emurgo/cross-csl-core'
-import {PoolInfoApi} from '@emurgo/yoroi-lib'
+import {FullPoolIfo, PoolInfoApi} from '@emurgo/yoroi-lib'
```

Comment:
it looks there is a typo in the lib

Reply:
created a PR: https://github.com/Emurgo/yoroi-lib/pull/157

Reply:
updated the version